esp/scm/ws_workunits_req_resp.ecm

```diff
@@ -1092,4 +1096,37 @@ ESPresponse [exceptions_inline] WUGetPluginsResponse
 {
     ESParray<ESPstruct WUEclPluginsInFolder> Plugins;
 };
+
+// ----------------------------------------------------------------------------------
+
+ESPrequest [nil_remove] WUAnalyseHotspotRequest
+{
+    string Wuid;
+    string RootScope;
+    bool OptOnlyActive(false);
+    bool OnlyCriticalPath(false);
+    bool IncludeProperties(true);
+    bool IncludeStatistics(true);
+    double ThresholdPercent(1.0);
+    [optional] ESPstruct WUPropertyOptions PropertyOptions;       // Which attribute details are returned
+};
+
+ESPStruct WUHotspotItem
+{
+    string Scope;
+    string ParentScope;
+    double TotalTimePercent;
+    double StartTimePercent;
+    double StartAndDependentsTimePercent;
+    double ExecuteTimePercent;
+};
```

Comment:
This doesn't appear to be returned in any response?

Reply:
I think it can be deleted.  This was before our discussion, and results now returned in wudetails format.